sirius-pipeline/bmv2/sirius_outbound.p4

```diff
@@ -53,7 +53,22 @@ control outbound(inout headers_t hdr,
             route_vnet; /* for expressroute - ecmp of overlay */
         }
 
-        counters = routing_counter;
+        counters = routing_v4_counter;
+    }
+
+    direct_counter(CounterType.packets_and_bytes) routing_v6_counter;
+
+    table routing_v6 {
```

Comment:
@prsunny Do we intend to have separate IPv6 and IPv4 tables or a joint table?

Reply:
We intend to have a joint table.

Reply:
@prsunny Why?  Most of the v4/v6 implementations I have seen for P4 models carry separate v4 and v6 table.  It is just a more cleaner design which allows a more efficient way to maintain/evolve/enhance the code.

Reply:
Table has to decide which key (example: IPv4 or IPv6).  Key construction, parsing the key, pass to table, table executes an action.  Split table to ensure separate buckets to parse (cleaner in one view).  
We have not seen (in SAI APIs) split tables.  We have a v6 and v4 table type in SAI.  

Do we want to have a unified SAI API?  Looking at the IP family will place an entry into the correct table (can place annotation on the table, for example bound to the same API).  Lists are a separate issue...

As an example:  UDP was anchored to same attribute in the metadata.

_After review of SAI, it looks like there is **1 table**, we can follow the model in SAI to remain consistent._

Reply:
@marian-pritsak @prsunny As discussed, I have made the changes to use one table for both v4 and v6 processing.  Please review the changes.  Thanks.

Reply:
@marian-pritsak can likely look at this soon and approve.  Marian would like to merge this first, prior to test infrastructure.

Reply:
Marian has a larger merge coming behind this.  This PR takes care of the routing, ACL will come later after Marian's check in  after the new method of creating the environment.